### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (p)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14490]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
.col-sm-12 > p
#### Snippet
`<p>
            When you build software with Bazel, you're running the same code that has been refined and tested for years at Google to build heavy-duty, mission-critical infrastructure, services, and applications.
          </p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.29 (foreground color: #777777, background color: #fafafa, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by fweikert** [comment]
We've migrated to a new site at https://bazel.build that fixes several accessibility issues.
